cluster-topology.md

```diff
@@ -0,0 +1,273 @@
+
+```sh
+kubectl --kubeconfig cluster-one.kubeconfig get nodes -w
+```
+
+Output:
+
+```
+NAME                                             STATUS     ROLES           AGE   VERSION
+docker-cluster-one-md-0-nrh7k-75ddc4778f-vlph9   Ready      <none>          86m   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   <none>          21s   v1.24.6
+docker-cluster-one-xbqb2-tlmpb                   Ready      control-plane   86m   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   <none>          21s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   control-plane   21s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   control-plane   23s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   control-plane   23s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   control-plane   30s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   control-plane   30s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   NotReady   control-plane   30s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   Ready      control-plane   31s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   Ready      control-plane   48s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   Ready      control-plane   48s   v1.24.6
+docker-cluster-one-xbqb2-8wcbj                   Ready      control-plane   53s   v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          0s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          0s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          0s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          0s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          2s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          2s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          7s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          7s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   NotReady   <none>          7s    v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   Ready      <none>          10s   v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   Ready      <none>          10s   v1.24.6
+docker-cluster-one-xbqb2-sjlwm                   Ready      <none>          12s   v1.24.6
+```
+
+## More scale operations
+
+For such a simple cluster topology change against a single configuration, it's also possible to update our Cluster resource in-place. Let's use `kubectl edit` to do that.
+
+```sh
+kubectl --kubeconfig kubecon-na-22-capi-lab.kubeconfig edit cluster/docker-cluster-one
```

Comment:
As mentioned above. I would drop the `--kubeconfig kubecon-na-22-capi-lab.kubeconfig` here (and in the other cases below)

Reply:
Adressed in follow-up